jumpscale/packages/vdc_dashboard/bottle/deployments.py

```diff
@@ -262,7 +262,11 @@ def accept():
 @app.route("/api/update", method="GET")
 @package_authorized("vdc_dashboard")
 def update():
-    branch = os.environ.get("SDK_VERSION", "development")
+    branch_param = request.params.get("branch")
```

Comment:
where do we send the branch ?

Reply:
we don't send it through UI. We said we will use it when requesting api on testing on another branches but user wouldn't switch branch